src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -160,6 +160,9 @@ def load_arguments(self, _):
                    help='A json file path for the persistent storages to be mounted to the app')
         c.argument('loaded_public_certificate_file', type=str, options_list=['--loaded-public-certificate-file', '-f'],
                    help='A json file path indicates the certificates which would be loaded to app')
+        c.argument('deployment', options_list=['--deployment', '-d'],
+                   help='Name of an existing deployment of the app. Default to the production deployment if not specified.',
+                   validator=fulfill_deployment_param_or_warning)
```

Comment:
I personally think it would be more appropriate to name it `deployment_name`, because this name can be consistent with the `az deployment` commands

Reply:
Here just update the validator for the argument. The name has been used for years.

Reply:
OK